powerlevel9k.zsh-theme

```diff
@@ -1317,8 +1317,8 @@ prompt_powerlevel9k_setup() {
 
   # Display a warning if the terminal does not support 256 colors
   local term_colors
-  term_colors=$(echotc Co)
-  if (( $term_colors < 256 )); then
+  term_colors=$(echotc Co 2>/dev/null)
+  if (( ! $? && ${term_colors:-0} < 256 )); then
```

Comment:
I think the test if the previous command (`echotc Co`) is unnecessary. A `print ${XXX_UNKNOWN:-0}` always returns zero, which is - surprisingly - always smaller than 256..